### ISSUE TITLE
[Feature] Update Closed Group Members list to show which member is the Group Admin

### ISSUE THREAD

**Issue by trav3ll3r** [oxen-io/session-android#1359]
### Is there an existing request for feature?

- [X] I have searched the existing issues

### What feature would you like?

Given the user is a member of a Closed Group
When the used navigates to the Closed Group and selects `Edit` option
Then the application shows the list of all Group Members in that Closed Group
And visibly presents a distinction between a Group Member and a Group Admin.

Currently on the `Edit Group` screen Group Members and Group Admins have exactly the same UI (Group Zombies are rendered differently)

It would be nice if this UI change could be added to the application.

**Comment by KeeJef** [comment]
Yeah this will be coming in the closed group overhaul task which we are progressing through now
